FVC-Observation-derivedFrom-example.js

```diff
@@ -0,0 +1,157 @@
+            }
+          ],
+          "text": "FVC Best (Pre-bronchodilator), liters"
+        },
+        "subject": {
+          "reference": "Patient/patient-01",
+          "display": "Example Patient Name"
+        },
+        "performer": [
+          {
+            "reference": "Practitioner/practitioner-01",
+            "display": "Dr. P. Pulmonologist"
+          },
+          {
+            "reference": "Organization/org-01",
+            "display": "Sample Pulmonary Function Laboratory"
+          },
+        ],
+        "valueQuantity": {
+          "value": 3.90,
+          "unit": "L",
+          "system": "http://unitsofmeasure.org",
+          "code": "L"
+        },
+       "referenceRange": [
+          {
+            "low": {
+              "value": 3.70,
+              "unit": "L",
+              "system": "http://unitsofmeasure.org",
+              "code": "L"
+              },
+            "high": {
+              "value": 5, // TODO: Placeholder value, not real ULN.
+              "unit": "L",
+              "system": "http://unitsofmeasure.org",
+              "code": "L"
+            },
+            "type": {
+              "text": "FVC (Best) normal bounds"
+            }
+          }
+        ],
+      }
+    },
+    {
+      "resource": {
+        "resourceType": "Observation",
+        "id": "ob-FVCpre-zscore",
+        "status": "final",
+        "code": {
+          "coding": [
+            {
+              "system": "http://loinc.org",
+              "code": "", // TODO: What code does z-score have? Same as FVC pre measurement?
+              "display": "", // TODO
+            }
+          ],
+          "text": "FVC Best (Pre-bronchodilator), z-score"
+        },
+        "subject": {
+          "reference": "Patient/patient-01",
+          "display": "Example Patient Name"
+        },
+        "performer": [
+          {
+            "reference": "Practitioner/practitioner-01",
+            "display": "Dr. P. Pulmonologist"
+          },
+          {
+            "reference": "Organization/org-01",
+            "display": "Sample Pulmonary Function Laboratory"
+          },
+        ],
+        "valueQuantity": {
+          "value": -1.34,
+          "unit": "", // TODO: How to code a z-score?
+          "system": "", // TODO
+          "code": "" // TODO
+        },
+       "referenceRange": [
+          // TODO: Think this might just be a string, "z-score"
+        ],
+        "derivedFrom": [
+          {
+            "reference": "Observation/ob-FVCpre-measurement"
+          }
+        ]
+      }
+    },
+    {
+      "resource": {
+        "resourceType": "Observation",
+        "id": "ob-FVCpre-percentpredicted",
+        "status": "final",
+        "code": {
+          "coding": [
+            {
+              "system": "http://loinc.org",
+              "code": "", // TODO: What code does an FVC % pred have? Same as FVC pre measurement?
```

Comment:
No not the same. 

Three things:

1. You can look them up when you know the code, for ex: https://loinc.org/19868-9/ - just to see what they mean etc
2. You can go to the other codebase - pft-extractor and reference them by the lookup mapping I've done previously (I think I showed you this?)
3. Richard (RIP, he died last year) was this guy who maintained a blog about PFTs and he posted his own LOINC mapping as an excel file in this post: https://www.pftforum.com/blog/loinc-and-why-it-matters-to-your-his-interface/

Reply:
Thanks, I was having trouble finding them on LOINC but the Excel sheet solved that right away.